tensorflow_encrypted/protocol/pond.py

```diff
@@ -787,18 +798,20 @@ def _lift(prot, x):
 
     raise TypeError("Don't know how to lift {}".format(type(x)))
 
-#
+
 # truncate
-# 
+# TODO
```

Comment:
?

Reply:
whoops misunderstood the comment below as the start of a new section